### PR TITLE
MGMT-8607: Store event names in the DB

### PR DESCRIPTION
**Pull request by nmagnezi**
# Assisted Pull Request

## Description

Events are a part of assisted-service API.
Event names will now be stored in db and included
in the events list API response.

## List all the issues related to this PR

- [x] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @filanov
/cc @masayag 
/cc @rollandf 

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by filanov** [comment]
LGTM, it will be nice to have another person to take a look

File: tools/generate_events.py [hunk 1]
```diff
@@ -81,7 +83,7 @@
 }
 
 func (e *{{eventName}}) GetName() string {
-    return "{{event['name']}}"
+    return e.EventName
```
Comment (masayag) [inline tools/generate_events.py:86]: my only comment is we might either delete the `GetName` function and use `e.EventName` directly
or change the `EventName` to be unexported and retain the `GetName`, assuming I haven't missed anything.
Having the `EventName` exported might let the caller the ability to modify the event-name which isn't desired.

Comment (nmagnezi) [reply tools/generate_events.py]: Changed it to unexported, to `eventName`

**Review by masayag — APPROVED** [review]

**Pull request merged**